code/addons/backgrounds/src/preview.tsx

```diff
@@ -1,8 +1,9 @@
+import type { Addon_DecoratorFunction } from '@storybook/types';
 import { withBackground } from './decorators/withBackground';
 import { withGrid } from './decorators/withGrid';
 import { PARAM_KEY } from './constants';
 
-export const decorators = [withGrid, withBackground];
+export const decorators: Addon_DecoratorFunction[] = [withGrid, withBackground];
```

Comment:
I would just use `import type { DecoratorFunction } from '@storybook/types';`

Reply:
These types are pretty much equal. When we do the types cleanup at 8.0 we'll sync this up.